TimeLog.API.Documentation/wwwroot/Reporting/GetContractPaymentsRaw.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tlp:ContractPayments 
+  xmlns:tlp="http://www.timelog.com/XML/Schema/tlp/v4_4"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.timelog.com/XML/Schema/tlp/v4_4 http://www.timelog.com/api/xsd/ContractPaymentsRaw.xsd">
+  <tlp:ContractPayment ID="23165" GUID="44ACE21C-5809-4D23-8A0F-7041D2DF8E1E">
+    <tlp:InvoiceDate>2011-06-04T00:00:00</tlp:InvoiceDate>
+    <tlp:Rate>0.0000</tlp:Rate>
+    <tlp:RateCurrency>0.0000</tlp:RateCurrency>
+    <tlp:Discount>0.000000000000000e+000</tlp:Discount>
+    <tlp:Quantity>1.000000000000000e+000</tlp:Quantity>
+    <tlp:Name>10. Forberedelse / Efterbehandling</tlp:Name>
+    <tlp:UnitType>0</tlp:UnitType>
+    <tlp:Amount>0.0000</tlp:Amount>
+    <tlp:AmountCurrency>0.0000</tlp:AmountCurrency>
+    <tlp:IsReadyForInvoicing>1</tlp:IsReadyForInvoicing>
+    <tlp:CustomerID>7968</tlp:CustomerID>
+    <tlp:CustomerName>Hempel A/S</tlp:CustomerName>
```

Comment:
Let's not use real company names in the samples.

Reply:
Adjustment made, thank you!